src/test/java/unit/ReporterTest.java

```diff
@@ -252,40 +254,40 @@ public void packageResultsTest() throws IOException {
     @Test
     public void packageResultsPositiveTest() throws IOException {
         String packageResults = null;
-        if (System.getProperty("packageResults") != null) {
-            packageResults = System.getProperty("packageResults");
+        if (System.getProperty(PACKAGE_RESULTS) != null) {
+            packageResults = System.getProperty(PACKAGE_RESULTS);
         }
         Reporter reporter =
                 new Reporter("results", "file", new Capabilities(new Browser("Chrome")), null, null, null, null, null, null);
         File directory = new File("results");
         File file = new File("results", "file");
 
-        System.setProperty("packageResults", "true");
+        System.setProperty(PACKAGE_RESULTS, "true");
         reporter.finalizeReporter(1);
         File results = new File("results", file.getName() + "_RESULTS.zip");
         assertTrue(results.exists());
 
         file.delete();
         results.delete();
         directory.delete();
-        System.clearProperty("packageResults");
+        System.clearProperty(PACKAGE_RESULTS);
         if (packageResults != null) {
-            System.setProperty("packageResults", packageResults);
+            System.setProperty(PACKAGE_RESULTS, packageResults);
         }
     }
 
     @Test
     public void packageResultsNegativeTest() {
         String packageResults = null;
-        if (System.getProperty("packageResults") != null) {
-            packageResults = System.getProperty("packageResults");
+        if (System.getProperty(PACKAGE_RESULTS) != null) {
```

Comment:
could we get a isPackageResultsSet too? Seems like the point of the Properties class.